src/main/java/network/brightspots/rcv/StreamingCVRReader.java

```diff
@@ -100,9 +102,11 @@
     // to keep our code simple, we convert 1-indexed user-supplied values to 0-indexed here
     this.firstVoteColumnIndex = source.getFirstVoteColumnIndex() - 1;
     this.firstVoteRowIndex = source.getFirstVoteRowIndex() - 1;
-    this.idColumnIndex = source.getIdColumnIndex() != null ? source.getIdColumnIndex() - 1 : null;
-    this.precinctColumnIndex =
-        source.getPrecinctColumnIndex() != null ? source.getPrecinctColumnIndex() - 1 : null;
+    this.idColumnIndex =
```

Comment:
same nit as above: I think it's generally easier to read `if` blocks and ternaries when the condition is affirmative. up to you, though

Reply:
I'll make a pass in the next PR to check for instances like this.